src/legacy/core_plugins/data/public/plugin.ts

```diff
@@ -122,6 +124,7 @@ export class DataPlugin implements Plugin<DataSetup, DataStart, {}, DataPluginSt
       ui: {
         SearchBar,
       },
+      filterTypes: esFilters.FILTERS,
```

Comment:
No need to re-export this static dependency from the shim.
You can import from the NP plugin directly

Reply:
Thanks, I have removed the changes to the data shim and now import directly from NP data plugin.

When we migrate maps to NP, will this import still work?

Reply:
Absolutely!
That's the NP way doing it (for static imports)